src/Notes/Notes.php

```diff
@@ -273,16 +269,11 @@ public static function possibly_delete_marketing_notes( $old_value, $value ) {
 	 */
 	public static function possibly_delete_survey_notes() {
 		$data_store = self::load_data_store();
-		$notes      = $data_store->get_notes(
-			array(
-				'type'   => array( Note::E_WC_ADMIN_NOTE_SURVEY ),
-				'status' => array( 'actioned' ),
-			)
-		);
+		$note_ids   = $data_store->get_note_ids_by_type( Note::E_WC_ADMIN_NOTE_SURVEY );
 
-		foreach ( $notes as $note ) {
-			$note = self::get_note( $note->note_id );
-			if ( $note ) {
+		foreach ( $note_ids as $note_id ) {
+			$note = self::get_note( $note_id );
+			if ( $note && ( $note->get_status() === Note::E_WC_ADMIN_NOTE_ACTIONED ) ) {
```

Comment:
Do we need these inner parenthesis?

Reply:
We don't, but I had to look that up just now, which goes to readability. 🤷